.abs/main.yaml

```diff
@@ -2,3 +2,4 @@ replace-chart-version-with-git: true
 generate-metadata: true
 chart-dir: ./helm/{APP-NAME}
 destination: ./build
+giantswarm-validator-ignored-checks: "C0001"
```

Comment:
The _helpers.tpl file is no longer under our control as it comes with the upstream chart and so the team label check fails even though it is present in _gs_helpers.tpl. This flag to abs disables this.